airbyte-integrations/connectors/destination-postgres/src/testFixtures/java/io/airbyte/integrations/destination/postgres/PostgresContainerFactory.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.postgres;
+
+import io.airbyte.cdk.testutils.ContainerFactory;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+public class PostgresContainerFactory implements ContainerFactory<PostgreSQLContainer<?>> {
```

Comment:
copying from slack dm - if these classes were copied out of source-postgres, let's add a comment explaining their origin